### DISCUSSION TITLE
how to achieve rtl in quarto visual editor?

### DISCUSSION THREAD

**Discussion by idshklein** [quarto-dev/quarto-cli#8199] [Q&A]
### Description

I'm writing a qmd document in hebrew. text is indented to the left in the visual editor, as such:
![image](https://github.com/quarto-dev/quarto-cli/assets/12258810/fe2f11b2-2ecf-47fd-a728-1d3982c438ff)
is it possible to change the text alignment to the right?

**Comment by cderv** [comment]
Thanks for reaching out. 

As this is a feature for the Visual Editor, I moved this to 
* https://github.com/quarto-dev/quarto/issues/349
